server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterHealthAction.java

```diff
@@ -36,28 +37,7 @@
 @ServerlessScope(Scope.INTERNAL)
 public class RestClusterHealthAction extends BaseRestHandler {
 
-    @Override
-    public List<Route> routes() {
-        return List.of(new Route(GET, "/_cluster/health"), new Route(GET, "/_cluster/health/{index}"));
-    }
-
-    @Override
-    public String getName() {
-        return "cluster_health_action";
-    }
```

Comment:
I would prefer to keep these two at the top of the class. I am not sure it's convention necessarily, but if you see the majority of the `BaseRestHandlers` have them on top, and it's very helpful to find this metadata at the top of the class.

I understand this is not how we structure classes in general, but in this case it is very helpful considering the nature of the class. Does this make sense to you?

I do not feel so strongly about `allowSystemIndexAccessByDefault`.